compiler/env/jittypes.h

```diff
@@ -149,13 +149,10 @@ typedef struct TR_InlinedCallSite
       #define FLUSH_MEMORY(smp)
    #endif
 #else
-   #if (__IBMC__ || __IBMCPP__)
-      void do_lwsync (void);
-      #pragma mc_func do_lwsync {"7c2004ac"}
-      #pragma reg_killed_by do_lwsync
-      #define FLUSH_MEMORY(smp) if( smp ) do_lwsync();
+   #if defined(__IBMC__) || defined(__IBMCPP__)
```

Comment:
Let's add
```
#if !defined(__ibmxl__)
#include <builtins.h>
#endif /* __ibmxl__ */
```
here as well. It must be getting included implicitly somehow for `__lwsync()` to work on old XLC, but it's better to be explicit.